contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/rest/webmvc/blueprint/DescriptionMessageSourceResolvable.java

```diff
@@ -0,0 +1,39 @@
+package com.contentgrid.spring.data.rest.webmvc.blueprint;
+
+import com.contentgrid.spring.data.rest.mapping.Property;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Value;
+import org.springframework.context.MessageSourceResolvable;
+import org.springframework.data.rest.webmvc.RootResourceInformation;
+
+@Value
+public class DescriptionMessageSourceResolvable implements MessageSourceResolvable {
+    String[] codes;
+
+    public static DescriptionMessageSourceResolvable forEntity(RootResourceInformation information) {
+        return forNestedProperty(information, List.of());
+    }
+
+    public static DescriptionMessageSourceResolvable forProperty(RootResourceInformation information, Property property) {
+        return forNestedProperty(information, List.of(property));
+    }
+
+    public static DescriptionMessageSourceResolvable forNestedProperty(RootResourceInformation information, Collection<Property> properties) {
```

Comment:
Descriptions of nested properties are not shown in json-schema when using this formula

Reply:
trying the following keys in `rest-messages.properties`:
```
rest.description.d\:invoice-item.document.filename=filename 0
rest.description.document.filename=filename 1
rest.description.content.filename=filename 2
rest.description.d\:document.filename=filename 3
rest.description.d\:content.filename=filename 4
```
Shows `filename 2` as description in json schema